src/quic/node_quic_util-inl.h

```diff
@@ -42,11 +42,12 @@ size_t QuicCID::Hash::operator()(const QuicCID& token) const {
   return hash;
 }
 
-bool QuicCID::Compare::operator()(
-    const QuicCID& lcid,
-    const QuicCID& rcid) const {
-  return lcid->datalen != rcid->datalen ?
-      false : memcmp(lcid->data, rcid->data, lcid->datalen) == 0;
+bool QuicCID::operator==(const QuicCID& other) const {
+  return memcmp(cid(), other.cid(), sizeof(ngtcp2_cid)) == 0;
+}
+
+bool QuicCID::operator!=(const QuicCID& other) const {
+  return !(*this == other);
```

Comment:
These weren't really related but since I was in here I went ahead and made the change